app/cdap/components/PipelineDiff/store/hooks.ts

```diff
@@ -13,15 +13,9 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
+import { useDispatch, useSelector } from 'react-redux';
+import { TypedUseSelectorHook } from 'react-redux';
+import { RootState, AppDispatch } from '../store';
 
-import styled from 'styled-components';
-
-const DiffListRoot = styled.div`
-  background: white;
-  border: 1px solid black;
-  height: 100%;
-  width: 400px;
-`;
-
-// TODO: CDAP-20688
-export const DiffList = DiffListRoot;
+export const useAppDispatch: () => AppDispatch = useDispatch;
```

Comment:
As part of your journey, consider write a shared doc about your learnings and findings with redux-toolkit. It will be really beneficial for the team